INSTALL.rst

```diff
@@ -43,17 +43,23 @@ Requirements
 Installation
 ------------
 
-First you need to install
-`pipenv <https://docs.pipenv.org/install/#installing-pipenv>`_, it will handle
-the virtual environment creation for the project in order to sandbox our Python
-environment, as well as manage the dependency installation, among other things.
-
-Create your working directory and ``cd`` into it. Clone inside this directory the project:
+First, create your working directory and ``cd`` into it. Clone inside this directory the project:
 
 .. code-block:: console
 
     $ git clone https://github.com/rero/rero-ils.git
 
+You need to install `pipenv`, it will handle the virtual environment creation for the project
+in order to sandbox our Python environment, as well as manage the dependency installation,
+among other things. A specific version of python and pipenv is needed:
+
+.. code-block:: console
+
+    $ pyenv install 3.6.9
```

Comment:
Is it `python 3.6.9` or any `python 3.6`?

Reply:
I think that any `python 3.6` should work, but `pyenv install` requires a specific version of python.